src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -618,11 +621,10 @@ export default class DefaultAudioVideoController implements AudioVideoController
       // To perform the "Reconnect" action again, the session should be in the "Connected" state.
       this.sessionStateController.perform(SessionStateControllerAction.FinishConnecting, () => {
         this.logger.info('failed to reconnect audio-video session');
-        this.handleMeetingSessionStatus(
-          new MeetingSessionStatus(
-            this.getMeetingStatusCode(error) || MeetingSessionStatusCode.TaskFailed
-          )
+        const status = new MeetingSessionStatus(
+          this.getMeetingStatusCode(error) || MeetingSessionStatusCode.TaskFailed
         );
+        this.handleMeetingSessionStatus(status, error.message);
```

Comment:
Do we have error.name here? If so, should we pass it in?

Reply:
That's a good call, I tested with manually throw errors, so error.name is just "Error" plus the error status code is added. I can add error.name to provide more info.

Reply:
Recommend passing an Error object instead of a string. See a comment above.

Reply:
Yep, sounds good, I'll update in the next revision